sheet_to_triples/tests/test_main.py

```diff
@@ -285,6 +285,8 @@ def test_run_runner_non_unique_from(self):
             args = main.parse_args(argv)
 
         runner = run.Runner.from_args(args)
+        # override default non_unique predicates from run.py with empty set
+        runner.non_unique = set()
```

Comment:
This seems like the wrong test change, if the generally expected result is actually:
```python
self.assertEqual(runner.non_unique, non_uniques.union(run._default_non_unique))
```